client/mochaConfig.js

```diff
@@ -0,0 +1,21 @@
+import { storiesOf, action, linkTo, describe, it, specs } from './.storybook/mockApi';
+import { expect } from 'chai';
+global.storiesOf = storiesOf;
+global.action = action;
+global.linkTo = linkTo;
+global.describe = describe;
+global.it = it;
+global.expect = expect;
+global.specs = specs;
+
+import { jsdom } from 'jsdom';
+
+global.document = jsdom('<!doctype html><html><body></body></html>', {
+ headers: {
+   'User-Agent':
+   'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_6_7)' +
+   ' AppleWebKit/534.24 (KHTML, like Gecko) Chrome/11.0.696.71 Safari/534.24'
+ }
+});
+global.window = document.defaultView;
```

Comment:
We also might need to change this to a version where window's properties are exposed.
( http://airbnb.io/enzyme/docs/guides/jsdom.html )
```
Object.keys(document.defaultView).forEach((property) => {
  if (typeof global[property] === 'undefined') {
    exposedProperties.push(property);
    global[property] = document.defaultView[property];
  }
});
```